src/contribution-guide/debugging.md

```diff
@@ -0,0 +1,5 @@
+# Debugging
+
+When you want to debug a WebAssembly binary, [wasminspect](https://github.com/kateinoigakukun/wasminspect) is very helpful to investigate.
+
+![demo](https://raw.githubusercontent.com/kateinoigakukun/wasminspect/master/assets/demo.gif)
```

Comment:
Separate `.md` files are rendererd as separate `.html` pages, right? I wonder if we could merge the contribution guide into one or two `.md` files for now, so that we don't have pages with only one or two paragraphs for easier navigation? Later when the contribution guide grows, we could split it further.